src/lib/app.js

```diff
@@ -116,6 +116,8 @@ class App {
 
     router.get('/transfers/:id', transfers.getResource)
     router.get('/transfers/:id/state', transfers.getStateResource)
+    router.get('/transfers/:id/hash_preimage', transfers.getPreimage)
+    router.get('/transfers/public_key', transfers.getPublicKey)
```

Comment:
This seems odd to me. Are there multiple keys? Even if there are it would seem more sensible to have an endpoint for getting keys and then have them listed based on what they're for.

Also, is it dangerous to make anyone rely on getting the public key over HTTPS? What if the signature scheme the ledger is using is more secure than TLS?